### PR TITLE
feature(style): simplify wallet view

### PR DESCRIPTION
**Pull request by gabaldon**
Add Send and Receive buttons and modals to simplify Wallet view.

### CODE REVIEW

File: src/components/Receive.vue [hunk 1]
```diff
@@ -1,71 +1,52 @@
...
-
-            <div class="submit">
-              <Button class="button" :onClick="generateAddress" type="primary"
-                >Generate address</Button
-              >
-            </div>
-          </div>
+      <div class="wrapper">
+        <div id="last-generated-address" class="address">
+          {{ addresses[addresses.length - 1].address }}
```
Comment (Tommytrg) [inline src/components/Receive.vue]: Move this to a computed property to be readable

File: src/components/Receive.vue [hunk 2]
```diff
@@ -1,71 +1,52 @@
...
-  },
   data() {
     return {
       label: '',
+      copied: false,
+      text: 'Copy',
     }
   },
   methods: {
+    copyAddress() {
```
Comment (Tommytrg) [inline src/components/Receive.vue]: IMHO, the copy logic could be moved to a copy function in utils and be called from here. 

```suggestion
    copyAddress() {
      const idToCopy = 'last-generated-address'
      this.copied = copy(idToCopy)
    }
```

File: src/components/Transactions.vue [hunk 3]
```diff
@@ -1,26 +1,127 @@
...
+          </button>
+          <button
+            @click="toogleDirectionRight(page)"
+            v-if="page < setPages.length"
+            class="page-link"
+          >
+            <font-awesome-icon class="icon-right" icon="angle-right" />
+          </button>
+        </div>
+      </div>
```
Comment (Tommytrg) [inline src/components/Transactions.vue]: Could this be moved to its own component? I.E. AddressListing.vue

File: src/components/Transactions.vue [hunk 4]
```diff
@@ -1,26 +1,127 @@
...
+        />
+        <div class="send-received">
+          <Button class="primary" :onClick="displayModalSend">
+            Send
+          </Button>
+          <Button class="secondary" :onClick="displayModalReceive">
+            Receive
+          </Button>
+        </div>
+        <el-dialog
```
Comment (Tommytrg) [inline src/components/Transactions.vue]: Could this  be moved to its own component? I.E. SendTransaction.vue

File: src/components/Transactions.vue [hunk 5]
```diff
@@ -1,26 +1,127 @@
 <template>
   <div class="transactions">
     <TransactionList class="list" :transactions="transactions" />
-    <Balances
-      :available="available"
-      :timelocked="timelocked"
-      :unconfirmed="unconfirmed"
-      :total="total"
-    />
+    <div class="col-right">
```
Comment (Tommytrg) [inline src/components/Transactions.vue]: Could you split this component in smaller ones?

File: src/components/Wallet.vue [hunk 6]
```diff
@@ -44,24 +38,23 @@ export default {
...
-          // receive: ['receiveError'],
-          send: ['createVTTError'],
-        },
-        {
-          getTransactionsError: this.getTransactionsError,
-          createVTTError: this.createVTTError,
-        }
-      )
-    },
+    // errors() {
```
Comment (Tommytrg) [inline src/components/Wallet.vue]: Remove comment

File: src/store/wallet.js [hunk 7]
```diff
@@ -87,6 +91,13 @@ export default {
       }
       state.addresses = Array.from(addressesMap.values())
     },
+    // setPages(state, { addresses }) {
```
Comment (Tommytrg) [inline src/store/wallet.js]: Remove comment

**Comment by gabaldon** [comment]
This PR fixed is included in #818

**Pull request closed without merging**